PyTado/http.py

```diff
@@ -89,7 +90,7 @@ def __init__(
         action: Action | str = Action.GET,
         payload: dict[str, Any] | None = None,
         domain: Domain = Domain.HOME,
-        device: int | None = None,
+        device: int | str | None = None,
```

Comment:
I've been wondering this myself. Why can it be an integer or string?

Reply:
no it's both ^^